frontend/src/components/ResetPassword.js

```diff
@@ -0,0 +1,109 @@
+import { useLocation } from "react-router-dom";
+import React, { useState } from "react";
+import buildPath from "./path";
+import { blankValidator} from "./Validators/InputValidator";
+import Modal from "../components/Modal";
+import { makePTag, makeActionButton, makeDiv, makeButton, makeLink, makeSpan, makeH2 } from "./divHelpers/divHelpers";
+
+function ResetPassword() {
+  const search = useLocation().search;
+  const userId = new URLSearchParams(search).get("userId");
+  const [errorMessage, setMessage] = useState("");
+  const [isOpen, setIsOpen] = useState(false);
+
+  function makeInputDiv(label, id, className, type) {
+    return (
+      <div className={className}>
+        <label className="form-label" htmlFor={id}>
+          {label}
+        </label>
+        <input className="form-control" type={type} id={id} />
+      </div>
+    );
+  }
```

Comment:
Instead of copy and pasting the function, import it from another file. Take a look at divHelpers

Reply:
i refactor and put that in divHelpers